### ISSUE TITLE
Add Bulk IP Addresses - VRF list empty

### ISSUE THREAD

**Issue by Chris-ZA** [digitalocean/netbox#806]
Hi,

I am getting an empty VRF drop down list when using the form to add **bulk** IP addresses.

The drop down list is populated fine when using the individual IP address form.  The list is only empty with the bulk form.

I'm running 1.8.1.

Thanks!

**Comment by jeremystretch** [comment]
I'm not able to replicate this. The form field is about as simple as it can be, so I'm not sure what could be going wrong:

```
vrf = forms.ModelChoiceField(queryset=VRF.objects.all(), required=False, label='VRF')
```

**Comment by Chris-ZA** [comment]
I figured out that this only happens when you don't have any VRF's defined.  I added a VRF and the list populates.  If you don't have any VRF's defined then the list doesn't get populated with the "Global" VRF but the data does get saved when the form submits.

Hope this helps?

**Comment by jeremystretch** [comment]
I've corrected the "none" label to read "Global" now (c264281), as it does for the single IP address form.
